apps/omg_eth/test/omg_eth/root_chain_test.exs

```diff
@@ -73,44 +70,42 @@ defmodule OMG.Eth.RootChainTest do
     # Skipping these specs for now as this function needs to be updated
     # to use the new ALD function (not hasToken?)
     @tag :skip
-    test "returns false if no token exists", %{contract: contract} do
+    test "returns false if no token exists", %{contracts: contracts} do
```

Comment:
these are all skipped and they're testing the rootchainhelper (wtf?) I think we can safely delete this. has_token doesn't even exist!